transport/src/main/java/io/netty/bootstrap/ServerBootstrap.java

```diff
@@ -146,14 +146,36 @@ public EventLoopGroup childGroup() {
         return childGroup;
     }
 
+    /**
+     * Return the configured {@link ChannelHandler} be used for the child channels or {@code null}
+     * if non is configured yet.
+     */
+    public ChannelHandler childHandler() {
+        return childHandler;
+    }
+
+    /**
+     * Return a copy of the configured options which will be used for the child channels.
+     */
+    public final Map<ChannelOption<?>, Object> childOptions() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1452)